modules/ui/src/main/resources/lucuma-css/lucuma-ui-table.scss

```diff
@@ -10,7 +10,8 @@
   // Cell 
   .p-datatable-thead>tr>th {
     white-space: nowrap;
-    overflow: overlay;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
Cool, I think it looked weird

Reply:
Definitely looks weird on headers. I'm leaving it for cells for the moment. We can always remove it later too if need be.